README.md

```diff
@@ -15,7 +15,23 @@ Be sure to have the following build tools installed before proceeding:
 ## Generate keystore for APK signing
 
 ```shell
+ # Generate and use Debug keystore for development and testing purposes
 keytool \
+ -genkey -v \
+ -storetype PKCS12 \
+ -keyalg RSA \
+ -keysize 2048 \
+ -validity 10000 \
+ -storepass 'android' \
+ -keypass 'android' \
+ -alias androiddebugkey \
+ -keystore android/app/debug.keystore \
+ -dname "CN=io.mosip.residentapp,OU=,O=,L=,S=,C=US"
```

Comment:
For release, find out the right registration details give for mosip domain and update.

Reply:
Since it has been created already and used for the first time. If we want to change the details we need to delete the app from play store and configure it again in the play console. How to proceed with this?

Reply:
We have re-configured the app with the keystore details and updated the same in the secrets.